modules/trusted-firmware-m/Kconfig

```
@@ -202,6 +211,12 @@ config TFM_ALLOW_NON_SECURE_RESET
 	  to issue a system reset of the chip without going through a secure
 	  service.
 
+config TFM_S_CODE_VECTOR_TABLE_SIZE
+	hex
+	default 0x144 if SOC_NRF9160
+	default 0x154 if SOC_NRF5340_CPUAPP
+	help
```

```suggestion
	default -1
	help
```

I wonder if this will silently become 0 and silently break null-pointer detection.

Perhaps -1 is more noisy.

---

-1 is not an acceptable number 0xffffffff will break it.